packages/agents/sequencer/src/lib/errors/relayer.ts

```diff
@@ -5,3 +5,9 @@ export class RelayerSendFailed extends NxtpError {
     super(`Relayer Send Failed`, context, RelayerSendFailed.name);
   }
 }
+
+export class NotEnoughRelayerFee extends NxtpError {
```

Comment:
We should specify in the error what the relayer fee provided was, which transfer this is regarding, and how much the relayer fee *should be* in order to be sufficient

Reply:
Alternatively we could just specify in the context object, that's fine

Reply:
yes it was actually being specified in the context object